tests/python/PyImportSortBearTest.py

```diff
@@ -20,4 +20,5 @@
     ('import xyz\nimport abc\n',
      'import abc\nimport xyz\n'),
     settings={'known_standard_library_imports': 'xyz',
-              'known_first_party_imports': 'abc'})
+              'known_first_party_imports': 'abc',
+              'known_third_party_imports': 'abc'})
```

Comment:
Can you explain this change to me? Tagging `abc` as both first party *and* third party seems meaningless. 

The `known_X_party_imports` params are used to override the group `iosort` would normally put these modules in. So to test `known_third_party_imports` you could for example import a standard library module and tag it as `known_third_party_imports`. I would do this in a separate `verify_local_bear`